cluster/kubevirtci.sh

```diff
@@ -15,22 +15,39 @@
 export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-'k8s-1.17'}
 
 KUBEVIRTCI_VERSION='0b941ea5dc647d3aea6f6a6fff95563d6ce0445e'
-KUBEVIRTCI_PATH="${PWD}/_kubevirtci"
+KUBEVIRTCI_REPO='https://github.com/kubevirt/kubevirtci.git'
+KUBEVIRTCI_PATH=${KUBEVIRTCI_PATH:-"${PWD}/_kubevirtci/cluster-up/"}
```

Comment:
We are not using this anywhere, are we?

Reply:
`KUBEVIRTCI_PATH` is used in `cluster`'s `kubevirtci.sh` and in `_kubevirtci` folder.

Reply:
Please add a comment noting that this variable is used only by vendored kubevirtci scripts. And make sure that we don't use in in any of our scripts.

Reply:
Done